### PR TITLE
Reacting with ct_semicolon to messages in introductions

### PR DESCRIPTION
**Pull request by arnavsirigere**

### CODE REVIEW

**Comment by GypsyDangerous** [comment]
so this would react to any message sent in introduction with ct_semicolon, which is waving.

File: src/constants.js [hunk 1]
```diff
@@ -5,5 +5,6 @@ exports.emojis = {
     photoblob: '<:photoblob:660161410042691615>',
     blobtea: '<:blobtea:660162050558787596>',
     ct_asterisk: '<:ct_asterisk:664125053008609300>',
+    ct_semicolon: '664125053058678814',
```
Comment (nobobo1234) [inline src/constants.js]: Please adhere to the structure of the other emojis, with the name in the text.

File: src/events/message.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+const { emojis } = require('../constants');
+
 exports.run = (bot, message) => {
     bot.commands.handleCommand(message);
+
+    if (message.channel.id === bot.config.introductionsChannel) {
+        message.react(emojis.ct_semicolon);
+    }
```
Comment (nobobo1234) [inline src/events/message.js]: I'd rather have this outsourced in the commands manager, make a separate class method e.g. `handleMessage` or `handleReactions` and call that from here, and before the Commands#handleCommand call.

**Review by nobobo1234 — CHANGES_REQUESTED** [review]
Here you go :)

**Comment by arnavsirigere** [comment]
Ok. I think i fixed it now :)

File: src/managers/commands.js [hunk 3]
```diff
@@ -76,6 +77,13 @@ class Commands {
         }
     }
 
+    handleReactions(msg) {
+        if (msg.channel.id === this.bot.config.introductionsChannel) {
+            const [ctSemicolon] = emojis.ct_semicolon.match(/\d+/);
```
Comment (nobobo1234) [inline src/managers/commands.js:82]: One small change I'd like to request for the sake of consistency, you don't have to get the id from the emoji itself, you are able to react with the full string that is present in the emoji.js file as I did [here](https://github.com/CodingTrain/trainbot/blob/master/src/commands/information/userinfo.js#L12-L15).

**Review by nobobo1234 — CHANGES_REQUESTED** [review]

**Comment by nobobo1234** [comment]
All the other stuff looked great!

**Comment by arnavsirigere** [comment]
Im pretty sure you get an error if you try to react with the full string. The react function expects only the id if im not wrong. Putting the full string does work in a message however

**Comment by nobobo1234** [comment]
You are absolutely right, I'll approve it :)

**Review by nobobo1234 — APPROVED** [review]

**Comment by nobobo1234** [comment]
@all-contributors please add @arnavsirigere for code

**Comment by nobobo1234** [comment]
@all-contributors please add @nobobo1234 for review

**Comment by arnavsirigere** [comment]
I just realised this isnt working. The bot is not reacting to messages. Any idea why?

**Comment by nobobo1234** [comment]
I didn't look closely when I pulled my changes, should be working now!

**Pull request merged**